lib/python/qmk/tests/test_cli_commands.py

```diff
@@ -58,13 +58,44 @@ def test_list_keyboards():
     assert 'handwired/onekey/pytest' in result.stdout
 
 
-def test_list_keymaps():
-    result = check_subcommand("list-keymaps", "-kb", "handwired/onekey/pytest")
+# list-keymaps CLI tests
+def test_list_keymaps_short():
+    # check short arg format
+    result = check_subcommand('list-keymaps', '-kb', 'handwired/onekey/pytest')
     assert result.returncode == 0
-    assert "default" and "test" in result.stdout
+    assert 'default' and 'rgb' in result.stdout
```

Comment:
```suggestion
    assert 'default' and 'test' in result.stdout
```

The `test` keymap is important here, because it's a community layout. (`layouts/community/ortho_1x1/test`)